test/test/when.cpp

```diff
@@ -5,7 +5,13 @@ using namespace concurrency;
 using namespace winrt;
 using namespace Windows::Foundation;
 
-struct CommaStruct : std::experimental::suspend_never
+#ifdef __cpp_lib_coroutine
+using std::suspend_never;
```

Comment:
Same here

Reply:
Adding extra indentation for the lines inside ifdef blocks also seems odd to me.

Reply:
Fine as well